### ISSUE TITLE
Could Not Load krom.js (Xcode)

### ISSUE THREAD

**Issue by alolalo** [armory3d/armortools#1450]
**ArmorPaint version: 0.8

**OS/device including version: MacOS Monteray using OCLP on an iMac 2011, GPU: AMD WX 4130 (Metal support). The GPU uses a custom BIOS to make it work on an iMac, but this shouldn't be a problem as it supports metal shaders.

**Issue description: After succesfully building the Xcode project, running it results in an error stating: Could not load krom.js, aborting. I checked if the file was there and it was, so I have no clue on why it not loads other than the message itself.

**Steps to reproduce: follow the installation described in the READ.me, then open the project in Xcode, build it and run it

**Comment by Scooby08** [comment]
I also have this issue. I’m on a 13 inch 2022 Intel MacBook Pro running Monterey

**Comment by Scooby08** [comment]
I was able to get it working my opening ArmorPaint.app directly. Just search for “ArmorPaint.app” in finder. You might need to enable unknown sources and right click open to get it working.

**Comment by alolalo** [comment]
None found on my end, so the .app is never made

**Comment by lylythechosenone** [comment]
Anything new on this? I have the same issue. I tried opening `ArmorPaint.app`, but it just crashes as soon as it opens.

**Comment by lylythechosenone** [comment]
Using the `--compile` option (copying from linux, but with `metal`) works fine, and the output app works.

**Comment by pogotc** [comment]
> Using the `--compile` option (copying from linux, but with `metal`) works fine, and the output app works.

Just to clarify this as I'm also having trouble compiling this on a Mac, are you running:

```
../armorcore/Kinc/make --from ../armorcore -g metal  --compile
```

or 

```
../armorcore/Kinc/make --from ../armorcore -g metal  --compiler clang --compile
```

neither works for me but I'm curious which one worked for you...

**Comment by StanleySweet** [comment]
Just compile worked for me, then I had to build in Release for the app to get outputted which I could eventually run. Now needs the fix at #1665 though.

**Comment by luboslenco** [comment]
Merging with https://github.com/armory3d/armortools/issues/1671, more fixes hopefully soon.
